### PR TITLE
PLATFORM BUS APP - Regression updates for Etracking, GCP and GDC Apps

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by Mariachaudhry — COMMENTED** [review]
Reviewed

File: src/test/java/appsCommon/Utils/ServiceNow_Common_Methods.java
Comment (iamdez99) [inline src/test/java/appsCommon/Utils/ServiceNow_Common_Methods.java:1]: Why are we changing 2 seconds to 5 seconds?

File: src/test/java/ServiceNow/PlatformBusinessApps/GDC/GDC_Runners/RunGDCRegressionTest.java
Comment (iamdez99) [inline src/test/java/ServiceNow/PlatformBusinessApps/GDC/GDC_Runners/RunGDCRegressionTest.java:1]: Please connect with me to update this runners class.

**Review by iamdez99 — CHANGES_REQUESTED** [review]

**Review by iamdez99 — APPROVED** [review]

**Review by katm007 — APPROVED** [review]
Reviewed and approved!

**Review by sarwarahmed1 — APPROVED** [review]
Approved!

**Pull request merged**